Essentials/src/com/earth2me/essentials/utils/MaterialUtil.java

```diff
@@ -25,9 +25,26 @@
     private static final Set<Material> SIGN_POSTS;
     private static final Set<Material> WALL_SIGNS;
 
+    private static final Set<Material> HELMETS;
+    private static final Set<Material> CHESTPLATES;
+    private static final Set<Material> LEGGINGS;
+    private static final Set<Material> BOOTS;
```

Comment:
I would rather separate these with commas into one field.
```suggestion
    private static final Set<Material> HELMETS, CHESTPLATES, LEGGINGS, BOOTS;
```

Reply:
Doesn't follow our codestyle, let alone the codestyle of this file.